### ISSUE TITLE
Run Swift Script command doesn't work on Windows

### ISSUE THREAD

**Issue by svanimpe** [swift-server/vscode-swift#423]
The "swift: Run Swift Script" command doesn't work on Windows, as is to be expected.
It shows the same error as when running `swift file.swift` on the command line:

```
<unknown>:0: error: could not load the swift standard library
```

Environment: Swift 5.7 on Windows 11

**Comment by dharmatech** [comment]
Also running into the same issue.

I can run `swiftc` successfully at a PowerShell prompt:

![image](https://user-images.githubusercontent.com/20816/218241857-8305f23c-8d7b-4b3a-a6f0-a0835f071336.png)

However, if I use the `Run Swift Script` command in vscode, I get the following:

![image](https://user-images.githubusercontent.com/20816/218241942-4808b4e3-e0f7-4699-8696-83433ed09e14.png)

**Comment by adam-fowler** [comment]
The last time I looked running swift scripts on Windows was broken ie running `swift <file>`. @compnerd should be able to give us the latest details on that. Maybe they are fixed in the 5.8 development build.

**Comment by dharmatech** [comment]
Just wanted to clarify.

Compiling with `swiftc` at a PowerShell prompt works fine and the resulting executable works:

![image](https://user-images.githubusercontent.com/20816/218644257-a048cfe1-30c0-42f2-ba3a-c586f119cf6c.png)

Here's the output from evaluating that code:

![image](https://user-images.githubusercontent.com/20816/218644420-8ae93b33-032c-417d-928b-f3525273b2e6.png)

However, running `test.swift` directly via `swift` at a PowerShell prompt

![image](https://user-images.githubusercontent.com/20816/218644552-105aba7f-5519-47c7-aaa9-c5e3cc9f8c43.png)

runs into the issue:

![image](https://user-images.githubusercontent.com/20816/218644611-2b5761b4-4cef-499b-959f-9250b7450c97.png)

The file `test.swift` contains:

````
print("abc")
````

**Comment by compnerd** [comment]
@dharmatech correct.  `swift ...` is running the swift file as a script, which runs it through the interpreter, which is using LLDB.  `swift` compiles the code, doesn't interpret it.

**Comment by Taschi120** [comment]
Do I understand the above comments correctly: Running Swift with LLDB is not currently possible on Windows *at all*, and Swift code thus cannot be debugged on Windows?

**Comment by adam-fowler** [comment]
> Do I understand the above comments correctly: Running Swift with LLDB is not currently possible on Windows *at all*, and Swift code thus cannot be debugged on Windows?

No this is about running swift as a script. You can use LLDB to debug Swift executables.

**Comment by award999** [comment]
Blocked on getting driver support by the looks of it https://github.com/swiftlang/swift-driver/issues/1182
